spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/SparkAtlasStreamingQueryEventTracker.scala

```diff
@@ -39,8 +39,6 @@ class SparkAtlasStreamingQueryEventTracker(
     this(new AtlasClientConf)
   }
 
-  private val streamQueryHashset = new mutable.HashSet[java.util.UUID]
```

Comment:
This is the key point of patch's changeset: we no longer evaluate streaming event only once, which would miss lately activated sources and sinks.